terraform/ec2/linux/install-efs-utils.sh

```diff
@@ -0,0 +1,32 @@
+# install amazon EFS client on linux, agnostic to yum, apt, or zypper + rpm
+# https://docs.aws.amazon.com/efs/latest/ug/installing-amazon-efs-utils.html
+
```

Comment:
Overall, the step for Suse and Debian/Ubuntu is to clone and install the package, while Amazon Linux does have a similar step with Suse without using zypper. If that's the case, should we apply the general installation for all OS and check if zypper is there, we would use zypper to refresh (to avoid OpenSuse case)?